File: JULIA/Cap05/c05ex05.jl
# c05ex05.jl

global i = 1
while (true)
  print("Entre um valor numerico: ")
  n = parse(Int, readline())

  r = n * 3

  println("Resultado = $r")
  println()

  global i += 1
  if (i > 5) break end
end

println()
print("Tecle <Enter> para encerrar... ")
readline()

File: CSHARP/Cap05/c05ex05.cs
// c05ex05.cs

using System;
using static System.Console;

class Program
{
    static void Main(string[] args)
    {
        int I, N, R;

        I = 1;
        do
        {
            Write("Entre um valor numerico: ");
            N = int.Parse(ReadLine());

            R = N * 3;

            WriteLine($"Resultado = {R}\n");

            I++;
        } while (!(I > 5));

        Write("\nTecle <Enter> para encerrar... ");
        ReadLine();
    }
}
